src/player-area.js

```diff
@@ -89,10 +33,11 @@ export const PlayerArea = () => {
       // hacky approach for centering the map initially
```

Comment:
This hack should get addressed by https://github.com/mnogueron/react-easy-panzoom/pull/11